test/performance/benchmarks_test.go

```diff
@@ -87,10 +83,26 @@ func runTest(t *testing.T, pacer vegeta.Pacer, saveMetrics bool) {
 		t.Fatalf("Error probing domain %s: %v", domain, err)
 	}
 
+	endpoint, err := resolveEndpoint(clients.KubeClient, domain, test.ServingFlags.ResolvableDomain,
+		pkgTest.Flags.IngressEndpoint)
+	if err != nil {
+		t.Fatalf("Cannot resolve service endpoint: %v", err)
+	}
+
+	url := endpoint
+	if !strings.HasPrefix(url, httpPrefix) {
+		url = fmt.Sprintf("%s%s", httpPrefix, url)
+	}
+
+	headers := make(map[string][]string)
```

Comment:
it annoys me this isn't `http.Header` in vegeta.

Reply:
It annoys me the tool is called Vegeta and not Goku. Or Goten. OR EVEN TRUNKS!?